### PR TITLE
Fixing missing info for some templates

### PR DESCRIPTION
**Pull request by christian-oreilly**
#### What does this implement/fix?
Fixing missing info for the 2mo and the 18mo templates.

Example of bug this solves:
```
mne.datasets.fetch_infant_template("18mo"
```
```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-7-4ce04742900f> in <module>
----> 1 mne.datasets.fetch_infant_template("18mo", subjects_dir=subjects_dir)

<decorator-gen-485> in fetch_infant_template(age, subjects_dir, verbose)

~/code/mne-python/mne/datasets/_infant/base.py in fetch_infant_template(age, subjects_dir, verbose)
     90         destination=subj_dir,
     91         url=_ORIGINAL_URL.format(subject=subject),
---> 92         hash_=orig_hashes[f'{subject}.zip'],
     93     )
     94     return subject

KeyError: 'ANTS18-0Months3T.zip'
```

### CODE REVIEW

File: mne/datasets/_infant/base.py [hunk 1]
```diff
@@ -60,12 +61,12 @@ def fetch_infant_template(age, subjects_dir=None, verbose=None):
...
     # Code used to create the lists:
     #
-    # $ for name in 2-0Weeks 1-0Months 3-0Months 4-5Months 6-0Months 7-5Months 9-0Months 10-5Months 12-0Months 15-0Months 18-0Months 2-0Years; do wget https://github.com/christian-oreilly/infant_template_paper/releases/download/v0.1-alpha/ANTS${name}3T.zip; done  # noqa: E501
+    # $ for name in 2-0Weeks 1-0Months 2-0Months 3-0Months 4-5Months 6-0Months 7-5Months 9-0Months 10-5Months 12-0Months 15-0Months 18-0Months 2-0Years; do wget https://github.com/christian-oreilly/infant_template_paper/releases/download/v0.1-alpha/ANTS${name}3T.zip; done  # noqa: E501
     # $ md5sum ANTS*.zip
     # $ python
     # >>> import os.path as op
     # >>> import zipfile
-    # >>> names = [f'ANTS{name}3T' for name in '2-0Weeks 1-0Months 3-0Months 4-5Months 6-0Months 7-5Months 9-0Months 10-5Months 12-0Months 15-0Months 18-0Months 2-0Years'.split()]  # noqa: E501
+    # >>> names = [f'ANTS{name}3T' for name in '2-0Weeks 1-0Months 2-0Months 3-0Months 4-5Months 6-0Months 7-5Months 9-0Months 10-5Months 12-0Months 15-0Months 18-0Months 2-0Years'.split()]  # noqa: E501
```
Comment (larsoner) [inline mne/datasets/_infant/base.py:69]: You need to actually execute this code for at least the `2-0Months` case and add the resulting file to the repo, should live in `mne/datasets/_infant/ANTS2-0Months2T.txt`

File: mne/datasets/_infant/base.py [hunk 2]
```diff
@@ -36,7 +37,7 @@ def fetch_infant_template(age, subjects_dir=None, verbose=None):
     Parameters
     ----------
     age : str
-        Age to download. Can be one of ``{'2wk', '1mo', '3mo', '4.5mo', '6mo',
+        Age to download. Can be one of ``{'2wk', '1mo', '2mo', '3mo', '4.5mo', '6mo',
```
Comment (larsoner) [inline mne/datasets/_infant/base.py:40]: line too long

File: mne/datasets/_infant/base.py (hunk 1 above)
Comment (christian-oreilly) [reply mne/datasets/_infant/base.py:69]: Done.

**Comment by larsoner** [comment]
Thanks @christian-oreilly !

**Pull request merged**